ops/deploy_remote.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 set -e
 
-IMAGE=${1:-nearprotocol/nearcore:0.1.3}
+IMAGE=${1:-nearprotocol/nearcore:0.1.4}
 PREFIX=${2:-testnet-${USER}}
```

Comment:
show we have a flag to run on/off main one?

Reply:
i dunno.. it's pretty uncommon. this will only be used for the main one if it has to be recreated